__tests__/lint/assertions-casing-camel-case-error/.redocly.yaml

```diff
@@ -0,0 +1,19 @@
+apis:
+  main:
+    root: ./openapi.yaml
+
+lint:
+  rules:
+    assertions:
+      - context:
+        - type: Operation
+          matchParentKeys: [get]
+        subject: Operation
+        property: operationId
+        message: Operation id should be canmelCase
+        casing: camelCase
+      - subject: Operation
+        property: operationId
+        message: Operation id should be camelCase
```

Comment:
Let's have different messages in this rule. Otherwise it's not clear in the test result.

I also think we should add some assertion id so it's easy to find it later from the error message.

What do you think?

Reply:
Will be like this (where `Hello world` is predefined assertion id, if no  assertion id then assertion #number)

![Screenshot 2022-03-30 at 16 54 29](https://user-images.githubusercontent.com/17987710/160852183-7d2e291a-0327-4eb7-b9d6-a247ba94aaa3.png)
![Screenshot 2022-03-30 at 16 51 06](https://user-images.githubusercontent.com/17987710/160852270-c275a6d6-3678-4353-af58-a5295d817fd7.png)

Reply:
Great!